pkg/operator/operator.go

```diff
@@ -111,6 +112,9 @@ func New(
 	}
 
 	deployInformer.Informer().AddEventHandler(optr.eventHandlerDeployments())
+	validatingWebhookInformer.Informer().AddEventHandler(optr.eventHandlerSingleton())
+	mutatingWebhookInformer.Informer().AddEventHandler(optr.eventHandlerSingleton())
+	featureGateInformer.Informer().AddEventHandler(optr.eventHandler())
```

Comment:
why is this `featureGateInformer.Informer().AddEventHandler` being sneaked in here?